index.d.ts

```diff
@@ -382,8 +382,18 @@ type OnwardsType = {
     trails: TrailType[];
     description?: string;
     url?: string;
+    typeOfOnwards: TypeOfOnwards;
```

Comment:
Oh man, naming these types of props is hard. I mean, it should just be called `type` but good luck with that!

So I'm not strongly against `typeOfOnwards` but thought it might be nice to bounce some ideas around as sometimes that ends up working well.

What about `context`? Or maybe `variant`? Ideally, there would be some business nomenclature that we could adopt.

Reply:
Yeah, makes sense. I've updated this to `OphanComponentName` to describe actually what it is - and also means people know what they're changing if they change it!